contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -1,68 +1,704 @@
+        doc: |
+          Manipulator for positioning of the sample.
+        temperature_sensor(NXsensor):
           exists: recommended
-    (NXprocess):
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+        sample_heater(NXactuator):
+          exists: optional
+          name:
+            exists: recommended
+          physical_quantity:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          heater_power(NX_FLOAT):
+          (NXpid):
+            exists: recommended
+            setpoint(NX_FLOAT):
+              exists: recommended
+        cryostat(NXactuator):
+          exists: optional
+          name:
+            exists: recommended
+          physical_quantity:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          (NXpid):
+            setpoint(NX_FLOAT):
+              exists: recommended
+        drain_current_amperemeter(NXsensor):
+          exists: optional
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [current]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+        sample_bias_voltmeter(NXsensor):
+          exists: recommended
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [voltage]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+        sample_bias_potentiostat(NXactuator):
+          exists: recommended
+          name:
+            exists: recommended
+          physical_quantity:
+            enumeration: [voltage]
+          type:
+            exists: optional
+          (NXpid):
+            exists: recommended
+            setpoint(NX_FLOAT):
+              exists: recommended
+        device_information(NXfabrication):
+          exists: recommended
+          vendor:
+            exists: recommended
+          model:
+            exists: recommended
+          identifier:
+      pressure_gauge(NXsensor):
+        exists: recommended
+        doc: |
+          Device to measure the gas pressure around the sample.
+        name:
+          exists: recommended
+        measurement:
+          enumeration: [pressure]
+        type:
+          exists: optional
+        value(NX_FLOAT):
+          unit: NX_PRESSURE
+          doc: |
+            In case of a single or averaged gas pressure measurement, this is the scalar gas pressure around
+            the sample. It can also be an 1D array of measured pressures (without time stamps).
+        value_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_PRESSURE
+            doc: |
+              In the case of an experiment in which the gas pressure changes and is recorded,
+              this is an array of length m of gas pressures.
```

Comment:
I would also remove the log variants from the appdef

Reply:
The disadvantage is that for pressure_gauge and flood_gun, the description of these logs is not defined anywhere else, so we would loose this information.